src/ResourceManager/Resources/Commands.Resources/Microsoft.Azure.Commands.Resources.dll-Help.xml

```diff
@@ -4829,6 +4846,28 @@ Resources</dev:code>
           </command:commandLine>
         </command:commandLines>
       </command:example>
+      <command:example>
+        <maml:title>--------------------------  Example 2: Use a custom template uri and parameter uri with Incremental mode --------------------------</maml:title>
+        <maml:introduction>
+          <maml:paragraph>PS C:\&gt;</maml:paragraph>
+        </maml:introduction>
+        <dev:code>PS C:\&gt;New-AzureRmResourceGroupDeployment -Name testDeployment -ResourceGroupName ContosoEngineering -TemplateUri https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-vm-simple-windows/azuredeploy.json -TemplateParameterUri https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-vm-simple-windows/azuredeploy.parameters.json -Mode Incremental -DeploymentDebugSetting All</dev:code>
```

Comment:
Lets break it down each param to new line - that looks quite easy to understand and view, there are other examples like this in the help file.